Eukalyptus-Dreizack/deepcity-framework
# Switch to monorepo setup

## Description  
Switch to a monorepo setup using lerna.
Update the build system to support the monorepo.
Add two packages as example on how packages should look like.

## Related issue  
#45 

## How has this been tested?  

- Check all commands carefully that are listed in the package.json under scripts and ensurer that the build system work properly.
- Check if the dependencies are installed in the right way.

## Motivation and context  
We need a structured setup for our project that can handle the complex requirements of a large code base.

## Additional context:  
This PR should be reviewed by 2 reviewers to ensure quality due to the complexity of the new system.
